packages/devtools/lib/src/timeline/cpu_profile_protocol.dart

```diff
@@ -161,8 +161,11 @@ class CpuStackFrame {
   }
 
   @override
-  String toString() {
-    return '$name ($sampleCount samples, '
-        '${percent2(cpuConsumptionRatio)})';
+  String toString({Duration duration}) {
+    String str = '$name ';
+    if (duration != null) {
+      str += '- ${msText(duration, fractionDigits: 2)} ';
```

Comment:
Add a TODO to use a number of fractionDigits that better reflects the actual resolution.
For example, for the screenshot given fractionDigits: 1 would be enough.
In other words, how long is a single sample? We not show precision more than 10X or maybe 100X the length of a single sample.

Reply:
done.